pipeline/filtering/filtering_utils.py

```diff
@@ -206,7 +206,7 @@ def make_map_wrapper(obs, split_labels, pix_type="hp", shape=None, wcs=None,
         assert nside is not None
 
     if hasattr(obs.preprocess, "noiseQ_mapmaking"):  # ISO v2
-        inv_var = 1 / obs.preprocess.noiseQ_mapmaking.white_noise ** 2
```

Comment:
Would this change make ISO v2 runs fail? If yes, can we first check `hasattr(obs.preprocess.noiseQ_mapmaking, "std")` and if that is false use `white_noise` for backwards compatibility?